docs/editor/artificial-intelligence.md

```diff
@@ -87,11 +79,82 @@ In the image above, Copilot presents three suggestions. You can accept the entir
 
 If you don't want to accept any of the suggestions, you can continue typing, and Copilot will continue providing suggestions as you work.
 
+## Getting the most out of Copilot
+
+There are things you can do to help ("prompt") Copilot give you the best possible suggestions. And the good news is that you are probably already doing right things since they help you and your colleagues understand your code.
```

Comment:
"Prompt engineering" is probably a bit over-used at this point, but it feels like a familiar term users might expect or want to scan/ctrl+f for. Maybe it's worth adding an explicit mention of it, either here or somewhere else in this section?

Reply:
I think you're right that we should use the full term "prompt engineering" somewhere in here to ride the wave. 
Per my previous comment, I don't really think setting context is prompt engineering (the VS Code engineering team is hard at work doing that) but maybe we can craft a paragraph similar to https://github.blog/2023-06-20-how-to-write-better-prompts-for-github-copilot/#whats-a-prompt-and-what-is-prompt-engineering

Reply:
Makes sense to me!